### DISCUSSION TITLE
Migrated from Camunda 7 to CIB Seven

### DISCUSSION THREAD

**Discussion by pallavbhandari** [cibseven/discussions#3] [Q&A] [answered]
I managed to migrate from Camunda 7 Springboot to CIB Seven. However, I ran into an error while opening tasklist. The error reads:  Failure: An exception occurred in the persistence layer. Please check the server logs for a detailed message and the entire exception stack trace. 

The log shows: Error querying database.  Cause: org.postgresql.util.PSQLException: ERROR: column res.task_state_ does not exist
  Position: 419
The error may exist in org/cibseven/bpm/engine/impl/mapping/entity/Task.xml
The error may involve org.cibseven.bpm.engine.impl.persistence.entity.TaskEntity.selectTaskByQueryCriteria-Inline
The error occurred while setting parameters

**Comment by OlegCIB** [comment, marked as answer]
Dear @pallavbhandari , thank you for your interest in CIB seven.

I assume, you are migrating from Camunda 7.21 to CIB seven 1.1, am I right?
Our current straightforward migration path is from Camunda 7.22 to CIB seven, so it would be easier if you've upgraded from 7.21 to 7.22, based on their manual.

But as far as I understand your problem, all you need to do is run the following database migration script:
https://github.com/cibseven/cibseven/blob/main/engine/src/main/resources/org/cibseven/bpm/engine/db/upgrade/postgres_engine_7.21_to_7.22.sql

**Reply by pallavbhandari** [reply]
Thank you for the prompt response @OlegCIB. It worked!

**Reply by OlegCIB** [reply]
Great to hear that!
